ui/src/pages/AllStories.tsx

```diff
@@ -0,0 +1,288 @@
+import Paper from "@material-ui/core/Paper";
+import {
+  createStyles,
+  Theme,
+  WithStyles,
+  withStyles,
+} from "@material-ui/core/styles";
+import Switch from "@material-ui/core/Switch";
+import TableCell from "@material-ui/core/TableCell";
+import clsx from "clsx";
+import * as React from "react";
+import {
+  AutoSizer,
+  Column,
+  Table,
+  TableCellRenderer,
+  TableHeaderProps,
+} from "react-virtualized";
+import useSWR from "swr";
+
+import { colors } from "../styles/colors";
+import { Story } from "../types";
+
+declare module "@material-ui/core/styles/withStyles" {
+  interface BaseCSSProperties {
+    flip?: boolean;
+  }
+}
+
+const styles = (theme: Theme) =>
```

Comment:
Could be changed to `styled-components `but I pretty much left the styling the same as the example.

Reply:
I think this is fine considering the demo you posted looks good and we are tight on time to finish the project. Correct me if I'm wrong, but we can still mix styled-components in with createStyles elements right?

Reply:
Yeah let's just escape hatch for now, we can come back to this.